R/carehomes_rt.R

```diff
@@ -224,6 +224,199 @@ carehomes_Rt <- function(step, S, p, prob_strain = NULL,
 }
 
 
+##' @title Performs draws from the generation time distribution
+##'
+##' @param p A single [carehomes_parameters()] object.
+##'
+##' @param n The number of draws to perform
```

Comment:
we have not typically documented internal functions like this. If you do do this, please add `@nord` to prevent the creation of files in `man`. However, you do this inconsistently - did you mean to export this?